reference/configuration/framework.rst

```diff
@@ -1372,6 +1372,14 @@ cache
 The service that is used to persist class metadata in a cache. The service
 has to implement the :class:`Symfony\\Component\\Validator\\Mapping\\Cache\\CacheInterface`.
 
+.. versionadded:: 2.8
+
```

Comment:
@WouterJ what is the progress on your bot checking such things automatically ? Having it running as a github hook on PRs would save time from reviewer by avoiding the need to report such mistakes manually